api/src/main/java/com/codeforcommunity/rest/subrouter/AuthRouter.java

```diff
@@ -57,6 +61,16 @@ private void registerLogoutUser(Router router) {
     logoutUserRoute.handler(this::handleDeleteLogoutUser);
   }
 
+  private void registerVerifySecretKey(Router router) {
```

Comment:
Where are these routes documented? Have you updated an API definition?

Reply:
see nicoburniske-patch-1 pr

Reply:
Added to nicoburniske-patch-1 pr